x-pack/plugins/apm/public/components/shared/time_comparison/get_comparison_enabled.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { CoreStart } from 'kibana/public';
+
+const enableComparisonByDefault = 'observability:enableComparisonByDefault';
```

Comment:
The other settings keys are in https://github.com/elastic/kibana/blob/edd53f316fee4dcb0a1ceb55f401a11924e1200e/x-pack/plugins/observability/common/ui_settings_keys.ts#L8-L9

Reply:
I cannot import `plugins/observability/common/ui_settings_keys.ts` from `apm/public` it throws an error.

Reply:
Ah, I see it's already there. Just also duplicated here.

Reply:
Here is the error when I import from `plugins/observability/common/ui_settings_keys.ts`:

```
ERROR in ./public/components/shared/time_comparison/get_comparison_enabled.ts
       │          Module not found: Error: import [../../../../../observability/common/ui_settings_keys] references a non-public export of the [observability] bundle and must point to one of the public directories: [public]
       │           @ ./public/components/shared/time_comparison/get_comparison_enabled.ts 7:0-97 13:49-74
       │           @ ./public/components/shared/time_comparison/index.tsx
       │           @ ./public/components/shared/search_bar.tsx
       │           @ ./public/components/app/backend_detail_overview/index.tsx
       │           @ ./public/components/routing/home/index.tsx
       │           @ ./public/components/routing/apm_route_config.tsx
       │           @ ./public/components/routing/app_root.tsx
       │           @ ./public/application/index.tsx
       │           @ ./public/plugin.ts
       │           @ ./public/index.ts
       │           @ /Users/cauemarcondes/elastic/kibana/node_modules/@kbn/optimizer/target_node/worker/entry_point_creator.js
```

Reply:
You need to export it like: https://github.com/elastic/kibana/blob/d38f4884ad1d0548c6480b914b0b590783286e98/x-pack/plugins/observability/public/index.ts#L26

Reply:
https://github.com/elastic/kibana/blob/d38f4884ad1d0548c6480b914b0b590783286e98/x-pack/plugins/observability/common/index.ts#L9